script/scanRear.sh

```diff
@@ -33,6 +34,7 @@ rm scan_pid
 #scan_cmd="scanimage -l 0 -t 0 -x 215 -y 297 --device-name=$device --resolution=$resolution --batch=$output_file"
 scan_cmd="scanimage -l 0 -t 0 -x 215 -y 297 --resolution=$resolution --batch=$output_file"
 
+#TODO: remove one, isn't this known at build time?
```

Comment:
Yes, should be known. Not sure why I built it like this...Maybe changed the image at some point...

Reply:
change to sleep